examples/hybrid/staggered_nonhydrostatic_model.jl

```diff
@@ -364,12 +364,12 @@ function default_remaining_tendency!(Yₜ, Y, p, t)
     end
     @. ᶠω¹² += ᶠcurlᵥ(ᶜuₕ)
 
-    # TODO: Modify to account for topography
-    @. ᶠu¹² = Geometry.Contravariant12Vector(ᶠinterp(ᶜuₕ))
-    @. ᶠu³ = Geometry.Contravariant3Vector(ᶠw)
+    @. ᶠu¹² = Geometry.project(Geometry.Contravariant12Axis(), ᶠinterp(ᶜuvw))
+    @. ᶠu³ = Geometry.project(Geometry.Contravariant3Axis(), ᶠinterp(ᶜuvw))
```

Comment:
```suggestion
    @. ᶠu³ = Geometry.project(Geometry.Contravariant3Axis(), C123(ᶠinterp(ᶜuₕ)) + C123(ᶠw))
```
This avoids interpolating `ᶠw` twice